_includes/default-header.html

```diff
@@ -23,6 +23,7 @@
             <div class="navbar-dropdown">
               <a class="navbar-item" href="{% link openseeds/index.md %}">Open Seeds</a>
               <a class="navbar-item" href="{% link nebula/index.md %}">Nebula</a>
+              <a class="navbar-item" href="{% link catalyst/cat-1/schedule.md %}">Catalyst</a>
```

Comment:
Why CAT-1 schedule page here and not an index page of catalyst or at least the index page of CAT-1?

Reply:
Follow-up question: why is there no `index.md` for catalyst?

Reply:
> Why CAT-1 schedule page here and not an index page of catalyst or at least the index page of CAT-1?

About that! We were having a conversation at the team meeting, and Yo suggested that we avoid duplicating the already-existing cloud.catalyst.io website. So, we opted for keeping one schedule page per cohort.

What do you think?

Reply:
> Follow-up question: why is there no `index.md` for catalyst?

For the same reason stated above. 🙏🏽

Reply:
If several cohorts are planned, how will they be linked from our website?
It might be good to have a `index.md` file for catalyst that is hidden for displaying the stats (projects, mentors, etc) per cohort as we do for Open Seeds

Reply:
> Then we should remove the index.md file in catalyst/cat-1 folder

I did actually consider that, but every program must have a cohort-specific subfolder (e.g ols-1, cat-1), because of the way the schedule page is populated. Please, have a look at [this line](https://github.com/NPDebs/open-life-science.github.io/blob/24c32458fc7b0890b6f53c8e4bba20e6c5f40264/_includes/overall-schedule.md?plain=1#L5) in the `overall_schedule` file. Without the `index.md` page(s), we will unavoidably run into errors. 👇 

```
  Liquid Exception: Could not find document 'catalyst/cat-1/index.md' in tag 'link'. Make sure the document exists and the path is correct. in catalyst/cat-1/schedule.md
```

Reply:
> ...It might be good to have a index.md file...

I also envisaged a time when we would need to create [a table](https://openlifesci.org/openseeds/#cohorts) like the one in the `/openseeds` page. As much as this is a critical feature in the catalyst sub-site infrastucture, it does bring us back to the subject of duplicating the other Catalyst website.

Maybe we can ask for input from Taj and Yo?

Reply:
> > Then we should remove the index.md file in catalyst/cat-1 folder
> 
> I did actually consider that, but every program must have a cohort-specific subfolder (e.g ols-1, cat-1), because of the way the schedule page is populated. Please, have a look at [this line](https://github.com/NPDebs/open-life-science.github.io/blob/24c32458fc7b0890b6f53c8e4bba20e6c5f40264/_includes/overall-schedule.md?plain=1#L5) in the `overall_schedule` file. Without the `index.md` page(s), we will unavoidably run into errors. 👇 
> 
> ```
>   Liquid Exception: Could not find document 'catalyst/cat-1/index.md' in tag 'link'. Make sure the document exists and the path is correct. in catalyst/cat-1/schedule.md
> ```
> 
> 

The section linking to the `index.md` should then probably be removed for catalyst if we do not want people to access it even from the schedule page

Reply:
Good call. 👍 
I used a conditional to hide these lines in catalyst.

```
{%- if program != 'catalyst' %}
During the program,
...
{%- endif%}
```

But there is also [another line](https://github.com/NPDebs/open-life-science.github.io/blob/24c32458fc7b0890b6f53c8e4bba20e6c5f40264/_includes/overall-schedule.md?plain=1#L38) in the tabular schedule that is pointing to the index page.

Reply:
Should we seek opinions from others to decide whether we need a "previous cohorts" section (i.e 'catalyst/index.md')?

Reply:
> Good call. 👍 I used a conditional to hide these lines in catalyst.
> 
> ```
> {%- if program != 'catalyst' %}
> During the program,
> ...
> {%- endif%}
> ```
> 
> But there is also [another line](https://github.com/NPDebs/open-life-science.github.io/blob/24c32458fc7b0890b6f53c8e4bba20e6c5f40264/_includes/overall-schedule.md?plain=1#L38) in the tabular schedule that is pointing to the index page.

We really need to discuss together if we really want to remove the `cat-1/index.md` page as it explains how cohorts are run, etc